opal/util/ethtool.h

```diff
@@ -0,0 +1,25 @@
+#ifndef OPAL_ETHTOOL_H
+#define OPAL_ETHTOOL_H
+
+#include "opal_config.h"
+
+#if !defined(HAVE_DECL_ETHTOOL_CMD_SPEED)
```

Comment:
Do these `ethtool_cmd_speed()` definitions need to be in this .h file?  I think they can probably go in `ethtool.c`, since we don't need anyone else to be able to call `ethtool_cmd_speed()`.